bika/lims/content/bikasetup.py

```diff
@@ -916,6 +904,16 @@ def isRejectionWorkflowEnabled(self):
         else:
             return False
 
+    def getRejectionReasonsItems(self):
+        """Return the list of predefined rejection reasons
+        """
+        reasons = self.getRejectionReasons()
```

Comment:
I see that you removed the fied `RejectionReasons` above. Where does no this `getRejectionReasons ` come from?

Reply:
There were two "RejectionReasons" schema fields:
https://github.com/senaite/senaite.core/pull/1309/files/0462fb2292c3f7d42a4e5c09ccd7c0cc18ae1c50#diff-0dde5125b635d6fb55f5372798dae571R801

https://github.com/senaite/senaite.core/pull/1309/files/0462fb2292c3f7d42a4e5c09ccd7c0cc18ae1c50#diff-0dde5125b635d6fb55f5372798dae571L561

I've removed one of them